docs/release-notes.rst

```diff
@@ -10,6 +10,77 @@
  Version 0.26
 **************
 
+Version 0.26.2
+==============
+
+**Release Date:** October 25, 2023
+
+Notice: The ``ruamel.yaml`` library's 0.18.0 release includes breaking changes that affect earlier
+versions of Determined. The failure behavior is that commands which emit YAML like ``det experiment
+config`` will emit nothing to ``stdout`` or ``stderr`` message but exit 1 (originating from the new
+``ruamel.yaml`` library). This release of Determined has included a ``ruamel.yaml<0.18.0``
+requirement, but older versions of Determined will also be affected, so users of older versions of
+Determined may have to manually downgrade ``ruamel.yaml`` if they observe this behavior.
+
+**New Features**
+
+-  Python SDK: various new features and enhancements. A few highlights are listed below.
+      -  Add support for downloading a zipped archive of experiment code
```

Comment:
Remove three spaces of indentation from all inner lists. These all currently specify [definition lists](https://restructuredtextmenu.readthedocs.io/en/latest/lists/definition.html).